packages/eds-core-react/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -533,38 +535,46 @@ function AutocompleteInner<T>(
   )
 
   const optionsList = (
-    <StyledList
-      {...getMenuProps(
-        {
-          ref: setContainerEl,
-          style: styles.popper as UseComboboxGetMenuPropsOptions['style'],
-          'aria-multiselectable': multiple ? 'true' : null,
-          ...attributes.popper,
+    <div
```

Comment:
I see you added a new wrapper div for the list, I assume this was because of conflicting props with downshift?

Reply:
Yes trying to combine those two was a no go. It is probably also better to just separate those conserns anyway with all the attributes each are adding and changing all the time

Reply:
Yeah, sounds like the best approach